app/utils/wikiQuery.tsx

```diff
@@ -76,3 +76,112 @@ export function isPartialContestantData(contestantRowData: ITableRowData): boole
+        }
+        let teamName = element.name || element.name2
+
+        if (status === null || status === undefined) {
+            throw new ReferenceError("Status is either null or undefined and it should not be")
+        }
+
+        let isParticipating = true
+        let eliminationOrder = 0
+        let isWinner = false
+
+        // for amazing-race
+        if (status.toLowerCase().includes('eliminated')) {
+            isParticipating = false
+            eliminationOrder = Number(status.match(/Eliminated (\d+)/i)![1])
+        } else if (status.toLowerCase().includes("third")) {
+            isParticipating = false
+            eliminationOrder = (contestantData.length/2) - 2
+        } else if (status.toLowerCase().includes("runners-up")) {
+            isParticipating = false
+            eliminationOrder = (contestantData.length/2) - 1
+        }
+        // for big-brother
+        else if (status.toLowerCase().includes("evicted")) {
+            isParticipating = false
+            eliminationOrder = Number(status.match(/EvictedDay (\d+)/i)![1])
+        } else if (status.toLowerCase().includes("expelled")) {
+            isParticipating = false
+            eliminationOrder = Number(status.match(/ExpelledDay (\d+)/i)![1]) // covers Luke getting booted
+        } else if (status.toLowerCase().includes("exited")) {
+            isParticipating = false
+            eliminationOrder = Number(status.match(/ExitedDay (\d+)/i)![1]) // covers Jared leaving after zombie twist
+        } else if (status.toLowerCase().includes("runner-upd")) { // added the d to distinguish from amazing-race
+            isParticipating = false
+            eliminationOrder = Number(status.match(/Runner-UpDay (\d+)/i)![1]+ ".5") // covers the final person plus adding .5 to distinguish from the last eviction
+        } else if (status.toLowerCase().includes("winner")) {
+            isWinner = true
+        }
+
+        if (eliminationOrder !== 0) {
+            // updatePreviousExitDay
+            previousExitDay = eliminationOrder
+        } else if (!isWinner) {
+            // if no eliminationOrder is found set it to the previous exitDay
+            isParticipating = false
+            eliminationOrder = previousExitDay
+            const foundContestant = contestants[contestants.length-1]
+            foundContestant.exitedDay = foundContestant.exitedDay + 0.5 // acounts for the default ordering where the person who come first was actuall evicted last
+        }
+
+        contestants.push({
+            teamName: teamName,
+            relationship: element.col2,
+            isParticipating,
+            exitedDay: eliminationOrder
+        })
+    })
+
+
+    const sortedContestants = contestants.sort(function(a, b){
+        return a.exitedDay-b.exitedDay
+    })
+
+    let eliminationOrderCounter = 1
+    const contestantsWithEliminationOrder = sortedContestants.map(function(contestant) {
+        let eliminationOrder = Number.MAX_VALUE
```

Comment:
I do believe this triggers a warning because this will set a showContestant's `eliminationOrder` (probably the winner) to a non '0' value but but also there `isParticipating` will also be true, which is a warning as described [here in the Team ctor](https://github.com/jjm3x3/AmazingRaceFantasy/blob/main/app/models/Team.tsx#L13)